lib/src/revset.pest

```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-non_period_identifier = @{ (ASCII_ALPHANUMERIC | "_" | "@" | "/")+ }
+non_final_identifier = @{ (ASCII_ALPHANUMERIC | "_" | "@" | "/")+ }
```

Comment:
Should this be 'non_terminal_identifier' or some other word?  'final' strictly implies that it's the end character, though I don't know that 'terminal' is much better.